pyproject.toml

```diff
@@ -50,6 +50,13 @@ mypy = "^1.11"
 build = "^1.2"
 twine = "^5.1"
 codespell = "^2.3"
+# scipy is here even though it's for trulens-feedback because `poetry lock`
+# doesn't incorporate the python version specific dependencies in
+# `./src/feedback/pyproject.toml` correctly for some reason.
+scipy = [
```

Comment:
is it just scipy causing this issue right now? I added similar python-specific versioning for `tiktoken`, `onnxruntime`, `numpy`, and `torch` for different packages

Reply:
yeah for some reason `scipy` is the only one causing an issue for me in another PR

1. `tiktoken` luckily has `0.7.0` satisfying all the requirements.
2. `onnxruntime` has `1.19.2` satisfying python 3.12 and below and 3.13 we don't actually test anything on so it's fine.
3. `numpy` does seem like it could be a problem as the version `poetry.lock` has chosen (i.e. `1.26.4` isn't supported for python 3.8 but the pipeline shows that it somehow installed `1.24.4` do to another dependency.
4. `torch` has `2.5.1`. satisfying everything.